build/cqs.tsv

```diff
@@ -175,11 +184,11 @@ org.apache.parquet:parquet-hadoop	1.9.0	compile
 org.apache.parquet:parquet-jackson	1.9.0	compile
 org.apache.thrift:libthrift	0.9.3	compile
 org.apache.xml:xml-commons-resolver	1.2	compile
+org.apache.yetus:audience-annotations	0.5.0	compile
```

Comment:
What is bringing this in?

Reply:
this is pulled in by hbase 2.2, the other version is pulled in by kudu

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21497

Reply:
approved